slices/crun.yaml

```diff
@@ -0,0 +1,19 @@
+package: crun
+
+essential:
+  - crun_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libcap2_libs
+      - libseccomp2_libs
+      - libsystemd0_libs
+      - libyajl2_libs
+    contents:
+      /usr/bin/crun:
+      /usr/lib/*-linux-*/libcrun.a:
```

Comment:
This file should go in a separate `libs` slice. The slice then needs to be listed as an essential of `bins`.